### DISCUSSION TITLE
Some formulas are not displayed in markdown on github

### DISCUSSION THREAD

**Discussion by Taka0007** [community/community#60965] [Programming Help]
### Body

I wrote the following markdown file for my math notes.
However, only the part 「 $d$ は $X$」 is not displayed as a mathematical expression.
Could you please tell me how to deal with this problem?

Written markdown
↓
## 距離空間

<br>
以下の3つの性質を満たすとき 「 $d$ は $X$ 上の距離である」という。<br>

(i)任意の $x,y \in X$ について $d(x,y) \geqq 0$ であり, $d(x,y) = 0$ となるのは $x=y$ であるときのみである。<br>
→距離は必ず $0$ より大きい値を取り距離が $0$ の場合は同じ点の時に限る

(ii)

(iii) 任意の $x,y,z \in X$ について $D(x,z)$

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by Hsin-Hung** [comment]
Try switching to the English keyboard when typing the $ sign. The $ sign from a Japanese keyboard might not work in your case.
